Cargo.toml

```diff
@@ -13,6 +13,7 @@ documentation = "https://docs.rs/proc-macro2-diagnostics"
 [features]
 default = ["colors"]
 colors = ["yansi"]
+nightly = ["dep:version_check"]
```

Comment:
I would expect a `nightly` feature to fail on stable `rustc`, so maybe this should be called `try_nightly`?

Motivation: if I see `cargo build --features nightly` I'm probably going to assume that the default `rustc` is nightly.

Reply:
Makes sense to me.